src/lib/Tests/Form/DataMapper/PolicyUpdateMapperTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Tests\Form\DataMapper;
+
+use eZ\Publish\API\Repository\Values\User\Limitation\ContentTypeLimitation;
+use eZ\Publish\Core\Repository\Values\User\PolicyUpdateStruct;
+use eZ\Publish\API\Repository\Values\Content\LocationCreateStruct;
+use EzSystems\EzPlatformAdminUi\Form\Data\Policy\PolicyUpdateData;
+use EzSystems\EzPlatformAdminUi\Form\DataMapper\PolicyUpdateMapper;
+use PHPUnit\Framework\TestCase;
+use EzSystems\EzPlatformAdminUi\Exception\InvalidArgumentException;
+
+class PolicyUpdateMapperTest extends TestCase
+{
+    /**
+     * @var PolicyUpdateMapper
+     */
+    private $mapper;
+
+    public function setUp()
+    {
+        $this->markTestSkipped();
+        $this->mapper = new PolicyUpdateMapper();
+    }
+
+    public function tearDown()
+    {
+        unset($this->mapper);
+    }
+
+    /**
+     * @dataProvider dataProvider
+     * @param array $properties
+     */
+    public function testMap(array $properties)
+    {
+        $data = $this->mapper->map($this->createStruct($properties));
+
+        $this->assertEquals($this->createData($properties), $data);
+    }
+
+    /**
+     * @dataProvider dataProvider
+     * @param array $properties
+     */
+    public function testReverseMap(array $properties)
+    {
+        $struct = $this->mapper->reverseMap($this->createData($properties));
+
+        $this->assertEquals($this->createStruct($properties), $struct);
+    }
+
+    public function testMapWithWrongInstance()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Argument \'value\' is invalid: must be instance of ' . PolicyUpdateStruct::class);
```

Comment:
Usually we use annotations for this, but AFAICS [Sebastian himself is now in favor](https://thephp.cc/news/2016/02/questioning-phpunit-best-practices) of what you've use, so probably we should as well change our approach. Collecting opinions now...

Reply:
I think we should follow the new recommendation, they bring some benefits